docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -198,14 +198,37 @@ PUT my_index
 PUT my_index/_doc/1
 {
   "name": {
-    "first":  "Alice",
-    "middle": "Mary",
-    "last":   "White"
+    "first":  "John",
+    "middle": "Winston",
+    "last":   "Lennon"
   }
 }
 --------------------------------------------------
 // CONSOLE
 
+Note that the `path_match` and `path_unmatch` parameters match on object paths
+in addition to leaf fields. As an example, indexing the following document will
+result in an error because the `path_match` setting also matches the object
+field `name.title`:
```

Comment:
May be just finish the sentence a little  why it results in an error, something like this: 
"..also matches the object field `name.title` which can't be indexed as text".

Reply:
👍